subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -449,6 +458,9 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 		/* No SyncInfo flag in primary channel PDU */
 
 		/* Tx Power flag */
+		if (pri_hdr_prev.tx_pwr) {
+			pri_dptr_prev++;
```

Comment:
For this, and line 488, I think a more explicit decrement is better, e.g. 
`pri_dptr_prev += sizeof(uint8_t`), so it is clear that we want to add/extract a uint8_t (tx power) value.